libs/platform/src/lib/components/search-input/search-input.component.scss

```diff
@@ -161,23 +161,90 @@
     }
 }
 
-.is-loading ::ng-deep .fd-button:before {
-    animation: sap-icon-spin 2s infinite linear;
-}
+/* LOADING SPINNER */
+$fd-spinner-height: 20px !default;
+$fd-spinner-width--bar: 2px;
+$fd-spinner-width--gutter: 1px;
+$_width: $fd-spinner-width--bar*4+$fd-spinner-width--gutter*3;
+$_height: $fd-spinner-height;
 
-.is-loading ::ng-deep .fd-input-group__addon:hover .fd-button:before {
-    content: "\e03e";
-    animation: none;
-}
+::ng-deep .search-input {
+
+    & .fd-button {
+        overflow: hidden;
+    }
+
+    .fd-spinner {
+        width: $_width;
+        height: $_height;
+
+        &::before,
+        &::after {
+            width: $fd-spinner-width--bar;
+            background-color: #0a6ed1;
+        }
+
+        .fd-spinner__body {
+
+            &::before,
+            &::after {
+                width: $fd-spinner-width--bar;
+                background-color: #0a6ed1;
```

Comment:
any reason why at some places background-colour is used like this `background-color: #0a6ed1;` but in places like line 224 it is `background-color: var(--fd-color-action-1, #0a6ed1);` being used as a fallback color? Just trying to understand.

Reply:
I would advice you if you set colors to use the css variables from https://github.com/SAP/theming-base-content/blob/master/content/Base/baseLib/sap_fiori_3/css_variables.css

Reply:
The CSS variables are applied further down in the file (line 218) starting with `@supports(--css: variables)`.

For IE, which does not support the `var()` syntax, you need to have a direct assignment, i.e.
```
            background-color: #0a6ed1;
```

For browser which can understand the `var()` syntax, we place the variable definition in the `@supports(--css: variables)` block. This variables block needs to be defined after (or below) the hardcoded definition, so that the "cascading" CSS variable rules can override.

Reply:
On second thought, I think I may try to import the SCSS color variable definitions from core, so that I don't have the hardcoded values here, and I still get IE compatible CSS after SCSS compilation.

Reply:
It looks like `fundamental-styles` doesn't distribute their SCSS source files, so I won't be able to import the SCSS color definitions.

Reply:
@KevinOkamoto fundamental-styles doesn't own any colors. They are coming from the theme package https://github.com/SAP/theming-base-content/blob/master/content/Base/baseLib/sap_fiori_3/css_variables.css

If you want to have IE support - you can do the same way as we do in fundamental-styles - with PostCSS to compile 2 rows:
```
  color: #32363a;
  color: var(--sapTextColor,#32363a);
```